examples/2d/sprite_scale.rs

```diff
@@ -0,0 +1,133 @@
+//! Shows how to use sprite scaling to fill and fit textures into the sprite.
+
+use bevy::prelude::*;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_systems(Startup, setup)
+        .run();
+}
+
+fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
+    commands.spawn(Camera2d);
+
+    let square = asset_server.load("textures/slice_square_2.png");
+    let banner = asset_server.load("branding/banner.png");
+
```

Comment:
There should be some example cases with a sprite from a texture atlas.

Reply:
> There should be some example cases with a sprite from a texture atlas.

It's important to note that the sprite sheet won't function correctly with scaling. Scaling completely overrides both the scaling and offset for the sprite, disregarding the `rect`.

Reply:
I just updated the documentation to properly notify a user that it couldn't be used for a sprite sheet. For now, I don't know how to easily add additional scaling of the sprite sheet to the desired rect